### DISCUSSION TITLE
Canon CR3 files

### DISCUSSION THREAD

**Discussion by spike-hodge** [drewnoakes/metadata-extractor#630] [General]
Are Canon CR3 files files supported with metadata-extractor? If not is there anything I (as a non-programmer) could do to help? e.g. supply sample images etc.

**Comment by drewnoakes** [comment]
This is being tracked in #374.

**Comment by baodoan2023** [comment]
any update for this topic @drewnoakes ? Thanks

**Reply by drewnoakes** [reply]
See the linked issue.
